alarm/PolicyManager2.js

```diff
@@ -2118,46 +2118,57 @@ class PolicyManager2 {
         case 'create':
           for (const rawPolicy of rawData) {
             const { policy, alreadyExists } = await this.checkAndSaveAsync(new Policy(rawPolicy));
-            let result = policy;
-            if (alreadyExists == 'duplicated') {
-              result = 'duplicated'
-            }
-            results[action].push(result);
+            results[action].push({
+              policy: policy,
+              error: alreadyExists == 'duplicated'
```

Comment:
try ... catch on every operation?
maybe we should even return `alreadyExists` as a rejection

Reply:
yes, may some policies update will succeed but some not. Need return failures and success

Reply:
I mean try/catch on calls to `checkAndSaveAsync` and `updatePolicyAsync`. No guarantee it'll return without exception.

Reply:
and is there a way for client to know if the create fails on duplication or other reason?